baseapp/block_gas_test.go

```diff
@@ -86,8 +86,8 @@ func TestBaseApp_BlockGas(t *testing.T) {
 		appConfig := depinject.Configs(makeTestConfig())
 
 		err = depinject.Inject(appConfig,
-			&bankKeeper,
```

Comment:
Please perhaps provide a comment as to why the ordering of this modules changes, is it just alphabetic ordering?